### ISSUE TITLE
Could not pass event PlayerMoveEvent to BanFromClaim v1.5.5

### ISSUE THREAD

**Issue by pumkinisawesome** [Baktus79/BanFromClaim#9]
Version 1.5.3 said it fixed the PlayerMoveEvent problem, but my console is being spammed with this:

```[Server thread/ERROR]: Could not pass event PlayerMoveEvent to BanFromClaim v1.5.5
java.lang.LinkageError: loader constraint violation: when resolving interface method 'com.griefdefender.api.claim.Claim com.griefdefender.api.claim.ClaimManager.getClaimAt(com.flowpowered.math.vector.Vector3i)' the class loader org.bukkit.plugin.java.PluginClassLoader @7c8c796d of the current class, no/vestlandetmc/BanFromClaim/listener/GDListener, and the class loader org.bukkit.plugin.java.PluginClassLoader @8ddeca3 for the method's defining class, com/griefdefender/api/claim/ClaimManager, have different Class objects for the type com/flowpowered/math/vector/Vector3i used in the signature (no.vestlandetmc.BanFromClaim.listener.GDListener is in unnamed module of loader org.bukkit.plugin.java.PluginClassLoader @7c8c796d, parent loader 'app'; com.griefdefender.api.claim.ClaimManager is in unnamed module of loader org.bukkit.plugin.java.PluginClassLoader @8ddeca3, parent loader 'app')
	at no.vestlandetmc.BanFromClaim.listener.GDListener.onPlayerEnterClaim(GDListener.java:39) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor845.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.5.jar:git-Paper-720]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-720]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-720]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1416) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.PacketPlayInFlying.a(SourceFile:122) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.PacketPlayInFlying$PacketPlayInPositionLook.a(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1265) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1258) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1219) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1133) ~[patched_1.16.5.jar:git-Paper-720]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:290) ~[patched_1.16.5.jar:git-Paper-720]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

Could you help me with how to fix this?

**Comment by Baktus79** [comment]
Which version of GriefDefender are you using?

**Comment by pumkinisawesome** [comment]
I have 1.5.11, which looks like it’s the latest version…

**Comment by Baktus79** [comment]
I will investigate this further. I use this daily on my own server and have not had this problem.

**Comment by Baktus79** [comment]
Now I have made some attempts myself in the hope of recreating the error you are experiencing with yourself. But here I think you have a plugin that creates conflict. Could you have given me a plugin list?

**Comment by CrispyDairy** [comment]
@pumkinisawesome could you share your plugin list? I have the same issue, and I want to see what same plugins we have

**Comment by pumkinisawesome** [comment]
Here’s our plugin list:

AdminFun, AdvancedRegionMarket, AngelChest, AsyncWorldEdit, AutoMessager, BetterRTP, BlueMap, BookRules, ChatFeelings, ClearLag, CommandPanels, CoreProtect, CraftableSaddle, CraftingStore, CreateYourOwnMenus, DeluxeMenus, DiscordBotAPI*, DiscordSRV, DiscordSRV-Staff-Chat, Essentials, EssentialsAntiBuild, EssentialsChat, EssentialsGeoIP, EssentialsProtect, EssentialsSpawn, F3NPerm, GoldenCrates, GriefDefender, GSit, GUIShop, Harbor, HolographicDisplays, ID-Library, InventoryRollback, InvSeePlusPlus, JoinCommands*, KeepDeathDrop, LastLoginAPI, LibsDisguises, LiteBans, LuckPerms, MarriageMaster, Multiverse-Core, Multiverse-Portals, NexEngine, OreAnnouncer, OreAnnouncerGUI, PlaceholderAPI, PlayerParticles, PlayTime*, ProtocolLib, Shop, SlashDiscord*, SuperVanish, TAB, TogglePvp*, TradeSystem, TrollingFreedom, Vault, Votifier, VotingPlugin, Vulcan, WorldEdit, WorldEditSUI, WorldGuard, WorldGuardExtraFlags

Hope this’ll help!

**Comment by CrispyDairy** [comment]
So, we found ot that pumkin is a friend of mine lol. I didn't know their github. We realised that we both downloaded bluemap at the same time. Thats about when the issue started happening if I recall correctly. So that might be the issue plugin

**Comment by Baktus79** [comment]
Very good. I'll take a look at it when I get home from vacation, so next week I will see what I can do.

**Comment by CrispyDairy** [comment]
Alright thanks :)

**Comment by CrispyDairy** [comment]
Any updates on this issue?

**Comment by Baktus79** [comment]
I started looking at this yesterday and have made a few changes. I have not tested myself, but if you want to see if the changes solved the problem then I can compile an edition for you.

**Comment by CrispyDairy** [comment]
I don't think I'm technically savvy enough for that lol. I'll just wait for updates :)

**Comment by Baktus79** [comment]
This issue is now solved in [version 1.5.6](https://github.com/Baktus79/BanFromClaim/releases/tag/1.5.6).
